src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -231,6 +231,12 @@ internal static partial class CSharpCodeStyleOptions
             "csharp_prefer_static_local_function",
             "TextEditor.CSharp.Specific.PreferStaticLocalFunction");
 
+        public static readonly Option2<CodeStyleOption2<bool>> PreferReadOnlyStruct = CreateOption(
+            CSharpCodeStyleOptionGroups.Modifier, nameof(PreferReadOnlyStruct),
+            CSharpIdeCodeStyleOptions.Default.PreferReadOnlyStruct,
+            "csharp_style_prefer_readonly_struct",
+            "TextEditor.CSharp.Specific.PreferReadOnlyStruct");
```

Comment:
If we don't expose in Tools → Options UI, the roaming location is not needed.

Reply:
This will have a tools|options setting.